.openpublishing.redirection.json

```diff
@@ -2780,9 +2780,234 @@
         },
         {
             "source_path": "docs/framework/xaml-services/{}-escape-sequence-markup-extension.md",
-            "redirect_url": "/dotnet/framework/xaml-services/escape-sequence-markup-extension",
+            "redirect_url": "/dotnet/desktop-wpf/xaml-services/escape-sequence-markup-extension",
+            "redirect_document_id": false
```

Comment:
why are you changing to false?

Reply:
Because I already have an article redirecting with id to the target. there can't be two, correct?

Reply:
Ah now I see you also changed the path and added a second one for the old redirect. This change is fine then.